README.md

```diff
@@ -81,4 +81,13 @@ npm run dev
 
 > If `ngrok` expires, update the callbacks in steps 4 and 7 with the new `ngrok_url`. You can learn more about [persisting ngrok tunnels longer (ngrok)](https://ngrok.com/docs/getting-started/#step-3-connect-your-agent-to-your-ngrok-account).
 
-11. Consult our developer documentation to [install and launch the app](https://developer.bigcommerce.com/api-docs/apps/quick-start#install-the-app).
+11. To create and run a production build, an optimized version of the code must be built and then run:
```

Comment:
@slsriehl would you mind taking a pass at this 🙏

Reply:
I'm a bit confused about why we're suddenly talking about production, since the readme up until this point is mostly focused on running locally 

Also, I think this is a bit awkward, because if you're following the guide, you'd run `ngrok http 3000` which uses up that port, so running `npm run start -p 3000` would clearly collide. 

We should either change that to point to somewhere else, or just instruct people to run `npm run dev` instead of start. Dev will fallback to a different port if the default is in use, whereas `start` will crash with `EADDRINUSE`

Reply:
@rtalvarez I believe `ngrok http 3000` proxies whatever process is running on port 3000; if I recall correctly, ngrok runs on one of the 4000 ports.

Reply:
@bc-chaz I agree with @rtalvarez that production info should be presented separately.

Reply:
> @rtalvarez I believe `ngrok http 3000` proxies whatever process is running on port 3000; if I recall correctly, ngrok runs on one of the 4000 ports.

Ahh of course! My bad

Reply:
@rtalvarez @slsriehl the main impetus behind this is from fly.io who mentioned that this lack of information was problematic when trying to start and containerize the sample app. BC is looking at fly.io as a replacement for Heroku so we would likely want to include this information